packages/core/src/payment/payment-strategy-type.ts

```diff
@@ -5,7 +5,6 @@ enum PaymentStrategyType {
     ADYENV3_GOOGLEPAY = 'googlepayadyenv3',
     AFFIRM = 'affirm',
     AFTERPAY = 'afterpay',
-    AMAZON = 'amazon',
     AUTHORIZENET_GOOGLE_PAY = 'googlepayauthorizenet',
     AMAZONPAYV2 = 'amazonpay',
```

Comment:
Should we rename this to `AMAZONPAY` now?

Reply:
Done.